Project.toml

```diff
@@ -6,6 +6,7 @@ version = "0.12.0"
 Clustering = "aaaa29a8-35af-508c-8bc3-b662a17a0fe5"
 DataStructures = "864edb3b-99cc-5e75-8d2d-829cb0a9cfe8"
 DataValues = "e7dc6d0d-1eca-5fa6-8ad6-5aecde8b7ea5"
+Distances = "b4f34e82-e78d-54a5-968a-f98e89d6e8f7"
```

Comment:
Needs compat bounds - I think "0.8" should be sufficient until the next release, where we could revisit the bound?

Reply:
I have no idea, really, so that seems safe enough to me. I'm not using exotic functionality, it would seem to me.